skltemplate/tests/test_template.py

```diff
@@ -1,9 +1,9 @@
 import pytest
 import numpy as np
 
+from sklearn.utils._testing import assert_array_equal
+from sklearn.utils._testing import assert_allclose
```

Comment:
```suggestion
from numpy.testing import assert_array_equal
from numpy.testing import assert_allclose
```

Reply:
Thanks for the suggestion, I didn't catch the recent change indeed. Changes done.